sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Release History
+## 5.1.0-beta.1 (Unreleased)
+
+## 5.0.0 (2020-07-27)
+- Re-release of version `1.0.1` with updated version `5.0.0`.
 
 ## 1.1.0-beta.1 (Unreleased)
```

Comment:
We should remove the `## 1.1.0-beta.1 (Unreleased)` entry since it'll never get released.

Reply:
Ahh. Thank you for that.